src/solver/conv_direct_naive_conv_bwd.cpp

```diff
@@ -40,6 +40,9 @@ bool ConvDirectNaiveConvBwd::IsApplicable(const ConvolutionContext& ctx) const
        miopen::IsDisabled(MIOPEN_DEBUG_CONV_DIRECT_NAIVE_CONV_BWD{}))
         return false;
 
+    if(!ctx.use_hip_kernels)
```

Comment:
In some cases Naive Solvers use ASM kernels (that shall be controlled by `use_asm_kernels`):
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/037e006c19be0d810c5cf1ced53b02e10d99702d/src/solver/conv_direct_naive_conv.cpp#L86-L95

Reply:
@carlushuang Thanks for fixing this!